src/test/scala/org/zalando/nakadi/client/KlientSpec.scala

```diff
@@ -515,3 +484,53 @@ class KlientSpec extends WordSpec with Matchers with BeforeAndAfterEach with Laz
     service.start()
   }
 }
+
+object KlientSpec extends WordSpecLike /*for 'info()'*/ with ShouldMatchers {
+
+  lazy val objectMapper = new ObjectMapper()
+    .configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)
+    .disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS)
+    .setPropertyNamingStrategy(PropertyNamingStrategy.CAMEL_CASE_TO_LOWER_CASE_WITH_UNDERSCORES)
+    .registerModule(new DefaultScalaModule)
+
+  val MEDIA_TYPE = "application/json"
+  val TOKEN = "<OAUTH Token>"
+  val HOST = "localhost"
+  val PORT = 8081
+
+  // Moved this below the test steps so that a human reader gets first at the tests. By wrapping this in a companion
+  // object, we clearly indicate its scope (and can make it 'private' if we want - though with tests that's less
+  // relevant than with main code). AKa280116
+
+  private
+  class TestListener extends Listener {
+    var receivedEvents = new AtomicReference[List[Event]](List())
+    var onConnectionClosed, onConnectionOpened, onConnectionFailed = 0
+
+    override def id = "test"
+
+    override def onReceive(topic: String, partition: String, cursor: Cursor, event: Event): Unit = {
+
+      // Note: Changing 'println' to 'info' is a functional change. What is the intended action here?
+      //    - for giving information about tests being run, 'info' (or warning, etc.) are right. Their output
+      //      is placed alongside the test pass/fail reports, in the right location. 'println', on the other hand,
+      //      causes output not aligned with test reporting. IF things are intended to be visible "right then",
+      //      using 'System.err.println' could be yet another choice. AKa280116
+
+      info(s"WAS CALLED [topic=$topic, partition=$partition, event=$event]")
```

Comment:
where does info come from? actually, it has no real meaning. I added this output while I was debugging an error and I forgot to remove it...